content/en/platform/corda/5.0-beta/developing/notaries/plugin-cordapps-notary.md

```diff
@@ -40,9 +40,13 @@ The source code for all of these modules can be found under the [notary-plugins
 
 ## Notary Server CPB
 
-The non-validating notary server CPB only contains CPKs that are produced by R3. Therefore, R3 also produces a standard non-validating notary server CPB, to make for an easier user experience, rather than requiring network operators to produce a CPB. This CPB is available in our standard R3 maven repositories, under `com.r3.corda.notary.plugin.nonvalidating:notary-plugin-non-validating-server`.
+The non-validating notary server CPB only contains CPKs that are produced by R3. Therefore, to improve the user experience, R3 also produces a standard non-validating notary server CPB. 
 
-This is an example Gradle configuration fragment, which will download this:
+### Downloading the CPB
+
+This CPB is available from our [GitHub release page](https://github.com/corda/corda-runtime-os/releases/).
+
+The following is an example Gradle configuration fragment that downloads this:
```

Comment:
We have a bit of a chicken and the egg problem here, as the releases page is currently empty. We need to check with @ronanbrowne when this will be published - I know there were plans for this to be done automatically.

Regardless, I suspect the configuration fragment is then no longer required. But we will need to update one of our in-house environments (I'm thinking CSDE) to use the new way of getting the artifact, so we can decide on the best way to document this.

Reply:
@tlawson3 just want to confirm that the CSDE will pick up the notary artifact in the new place?
I don't think this affects the CSDE section of the docs.

Reply:
CSDE is aware, we have a change going in as part of Beta3 CORE-9853
As this change wont affect user's experience of the CSDE, I don't feel we need to mention it within the CSDE docs.